camille/inputs/bazefetcher.py

```diff
@@ -0,0 +1,41 @@
+from pandas import DataFrame, Timestamp, read_json
+from os import path, walk, listdir
+import dateutil
+
```

Comment:
I suggest adding some logging as well. Add this to here, and add some log-messages in strategic places using "logger.debug()" (or warning or whatever):

```python
import logging

logger = logging.getLogger(__name__)